static/src/javascripts/projects/common/modules/commercial/user-features.ts

```diff
@@ -368,8 +358,7 @@ const fakeOneOffContributor = (): void => {
 	});
 };
 
-const isAdFreeUser = (): boolean =>
-	isDigitalSubscriber() || (adFreeDataIsPresent() && !adFreeDataIsOld());
+const isAdFreeUser = (): boolean => isDigitalSubscriber();
```

Comment:
We should keep the check here for adFreeDataIsPresent(). This check looks for the ad free cookie, and isDigitalSubscriber() looks for the subscriber cookie.
```suggestion
const isAdFreeUser = (): boolean => isDigitalSubscriber() || adFreeDataIsPresent();
```